code/modules/clothing/gloves/color.dm

```diff
@@ -130,21 +130,33 @@
 
 /obj/item/clothing/gloves/color/latex
 	name = "latex gloves"
-	desc = "Cheap sterile gloves made from latex."
+	desc = "Cheap sterile gloves made from latex. Transfers minor paramedic knowledge to the user via budget nanochips."
 	icon_state = "latex"
 	item_state = "lgloves"
 	siemens_coefficient = 0.3
 	permeability_coefficient = 0.01
 	transfer_prints = TRUE
 	resistance_flags = NONE
 
+/obj/item/clothing/gloves/color/latex/equipped(mob/user, slot)
+	ADD_TRAIT(user, TRAIT_QUICK_CARRY, CLOTHING_TRAIT)

```

Comment:
I don't know why you changed this, I don't think it will work properly at all now

Reply:
what's wrong?

Reply:
you still need to check for `slot == SLOT_GLOVES` else it'll be added if you're holding them in your hands rather than wearing them, also

Reply:
aaaaaa

Reply:
> you still need to check for `slot == SLOT_GLOVES` else it'll be added if you're holding them in your hands rather than wearing them, also

did i fix it